.rubocop_todo.yml

```diff
@@ -1,23 +1,19 @@
 # This configuration was generated by
 # `rubocop --auto-gen-config`
-# on 2022-11-26 07:45:38 UTC using RuboCop version 1.22.3.
+# on 2025-03-31 23:04:51 UTC using RuboCop version 1.75.1.
 # The point is for the user to remove these configuration records
 # one by one as the offenses are removed from the code base.
 # Note that changes in the inspected code, or installation of new
 # versions of RuboCop, may require this file to be generated again.
 
-# It may be possible for us to use safe_load, but we'd have to pass the
-# safelists, like `whitelist_classes` into our serializer, and the serializer
-# interface is a public API, so that would be a breaking change.
-# Offense count: 13
-# Cop supports --auto-correct.
-Security/YAMLLoad:
+# Offense count: 1
+# This cop supports unsafe autocorrection (--autocorrect-all).
+Rails/ActiveSupportOnLoad:
   Exclude:
-    - 'lib/paper_trail/serializers/yaml.rb'
-    - 'spec/models/book_spec.rb'
-    - 'spec/models/gadget_spec.rb'
-    - 'spec/models/no_object_spec.rb'
-    - 'spec/models/person_spec.rb'
-    - 'spec/models/version_spec.rb'
-    - 'spec/paper_trail/events/destroy_spec.rb'
-    - 'spec/paper_trail/serializer_spec.rb'
+    - 'lib/paper_trail/frameworks/active_record.rb'
+
+# Offense count: 1
+# This cop supports unsafe autocorrection (--autocorrect-all).
+Rails/ApplicationRecord:
```

Comment:
This should be not in the todo file, because its not something that should be done (fixed). Can be disabled just inline in that file.